vss-extension.json

```diff
@@ -3,7 +3,7 @@
   "manifestVersion": 1,
   "id": "vercel-deployment-extension",
   "name": "Vercel Deployment Extension",
-  "version": "1.5.1",
+  "version": "1.3.1",
```

Comment:
You probably want to release this as 1.5.2 or 1.6.0 to make sure pipelines actually pick the fixed version

Reply:
Ah, yes, thanks @phidol. I'll make this `1.6.0`.